test/integration_test/test-deploy.sh

```diff
@@ -129,6 +136,10 @@ else
 	sed -i 's/'enable_cluster_domain'/'""'/g' /testspecs/stork-test-pod.yaml
 fi
 
+if [ "$storage_provisioner" != "portworx" ] ; then
```

Comment:
You shouldn't use the `if` here, just always set it to the variable, otherwise, if it is set to `portworx` it'll end up not updating the spec

Reply:
Fixed.